src/lauxlib.js

```diff
@@ -410,9 +410,18 @@ if (typeof window === "undefined") {
             this.f = null;  /* file being read */
             this.buff = new Buffer(1024);  /* area for reading file */
             this.pos = 0;  /* current position in file */
+            this.binary = false;
         }
     }
 
+    const toDataView = function(buffer) {
```

Comment:
Should make this node.js conditional (as it's particular to node.js's Buffer implementation)